### PR TITLE
Upgrade gnupg21 to 2.1.10

### PR DESCRIPTION
**Pull request by jonnybarnes**
I’m not sure about bottling, so those lines were removed

### CODE REVIEW

File: gnupg21.rb [hunk 1]
```diff
@@ -1,16 +1,9 @@
 class Gnupg21 < Formula
   desc "GNU Privacy Guard: a free PGP replacement"
   homepage "https://www.gnupg.org/"
-  url "https://gnupg.org/ftp/gcrypt/gnupg/gnupg-2.1.9.tar.bz2"
-  mirror "https://www.mirrorservice.org/sites/ftp.gnupg.org/gcrypt/gnupg/gnupg-2.1.9.tar.bz2"
-  sha256 "1cb7633a57190beb66f9249cb7446603229b273d4d89331b75c652fa4a29f7b6"
-
-  bottle do
```
Comment (dunn) [inline gnupg21.rb]: The test-bot handles the bottle block, so you can leave it.

**Comment by jonnybarnes** [comment]
The bot is failing to build the bottles because it can’t find gpg-zip but its removed by the formula.

**Comment by dunn** [comment]
Then you can safely remove line 81 and the comment above it; thanks!

**Comment by DomT4** [comment]
Refusing to build on Mavericks:
```
Undefined symbols for architecture x86_64:
  "_sqlite3_errstr", referenced from:
      _sqlite3_stepx in sqlite.o
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make[2]: *** [gpg2] Error 1
make[2]: *** Waiting for unfinished jobs....
make[1]: *** [all-recursive] Error 1
make: *** [all] Error 2
```

**Comment by jonnybarnes** [comment]
Doing this properly as oppose to via GitHub.

**Pull request closed without merging**